index.html

```diff
@@ -0,0 +1,107 @@
+<div class="main">
```

Comment:
I think you forgot some important called standard, every html document start with this tags and <html> tags. Please add this to your document.
<!DOCTYPE html
       PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
       "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

Reply:
- [ ] Good job using polite comments as suggestions instead of giving orders. But I think you could give a better explanation to the code owner because this comment is hard to understand. I think you are referring to the [boilerplate HTML](https://www.freecodecamp.org/news/basic-html5-template-boilerplate-code-example/). Please, give a better explanation of your suggestion :raised_hands:
- [ ] [OPTIONAL] As you can see in the bullet above, it could be a good idea to provide [links](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) for more explanation about what you are referring to :wink:
- [ ] [OPTIONAL] Also, I think it is a good idea to use emojis to make communication more human :smiley:

Reply:
Hello @oliverSCZ  you are doing a great job:

- There are no linter errors
- You documented your work in a professional way.
- And excellent work with your Readme.md file.

We believe that you can improve and be better every time, that is why I need you to make the following changes:

1. Great Job with your Index your desire and interest is noticeable, however following the instructions of [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) is necessary that you begin with the next `<html>` tags 

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>HTML 5 Boilerplate</title>
    <link rel="stylesheet" href="style.css">
  </head>
  <body>
	<script src="index.js"></script>
  </body>
</html>
```
if you forgot this rule, you can check again in this link [boilerplate HTML](https://www.freecodecamp.org/news/basic-html5-template-boilerplate-code-example). These tags are required at the beginning of your html file.😉

2. Another point, continuing with the instructions of [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) we can consider  use **article** tags `<article>` instead of `<div class="card">`. This will make your code more readable to other code partners 🙌
3. In your style.css file, you can use Flex Box in your `<ul>` tags. Remember that practice makes perfect... 🥇